web/src/user_group_edit.ts

```diff
@@ -1510,14 +1526,21 @@ export function maybe_reset_right_panel(groups_list_data: UserGroup[]): void {
     }
 }
 
+function realm_has_deactivated_user_groups(): boolean {
+    const realm_user_groups = user_groups.get_realm_user_groups(true);
+    const deactivated_group_count = realm_user_groups.filter((group) => group.deactivated).length;
+
+    return deactivated_group_count > 0;
+}
+
 export function update_empty_left_panel_message(): void {
     // Check if we have any groups in panel to decide whether to
     // display a notice.
     let has_groups;
     const is_your_groups_tab_active =
         get_active_data().$tabs.first().attr("data-tab-key") === "your-groups";
     if (is_your_groups_tab_active) {
-        has_groups = user_groups.get_user_groups_of_user(people.my_current_user_id()).length;
+        has_groups = user_groups.get_user_groups_of_user(people.my_current_user_id(), true).length;
```

Comment:
I think this change can be avoided in this commit as last commit has other changes for this. And not passing `true` in the below call to `get_realm_user_groups` makes this change incomplete.